packages/core/scss/components/form/_input.scss

```diff
@@ -111,10 +111,10 @@
 }
 
 @mixin element-textarea {
-  & {
```

Comment:
Removing this selector will affect selector specificity causing the wrong padding to get applied.